src/utils/validation.js

```diff
@@ -0,0 +1,21 @@
+const validators = {
+  email: (email) => {
+    return /\S+@\S+/.test(email)
+  },
+  citationUrl: (string) => {
+    let url
+    try {
+      url = new URL(string)
+    } catch (_) {
+      return false
+    }
+    if (url) return true
```

Comment:
Yeah... Linter gets angry when you don't assign return value of `new` instantiation to a variable, and then gets angry again if you don't reference a declared variable. Hence the check, it's kind of silly but functionally it should work fine. I don't think the `null` example will ever be a case. If it can't create a url, it will throw an error and fail validation.

I'm not sure if there's a good way to use the `URL` class without new. And other validation methods I checked relied on regex, which is fine, but I thought using `URL` is a bit cleaner to read.